usr/share/rear/lib/global-functions.sh

```diff
@@ -863,6 +863,62 @@ function umount_mountpoint_lazy() {
     umount $v -f -l "$mountpoint" >&2
 }
 
+# Unmount mountpoint $1 first with sleep and retry then with lazy
+# cf. https://github.com/rear/rear/pull/2909
```

Comment:
Shouldn't that refer to this PR (https://github.com/rear/rear/pull/3408) instead?

Reply:
https://github.com/rear/rear/pull/2909
is the original implementation background discussions
wherefrom this one is basically only a copy.